clang/test/Driver/fortran-preprocessor.f90

```diff
@@ -2,7 +2,7 @@
 
 ! -cpp should preprocess as it goes, regardless of input file extension
 ! RUN: %flang -cpp -c -DHELLO="hello all" -### %s 2>&1 | FileCheck %s --check-prefixes=ALL,CPP,PP
-! RUN: %flang -cpp -c -DHELLO="hello all" -### -c f95-cpp-input %s 2>&1 | FileCheck %s --check-prefixes=ALL,CPP,PP
+! RUN: %flang -cpp -c -DHELLO="hello all" -### -x f95-cpp-input %s 2>&1 | FileCheck %s --check-prefixes=ALL,CPP,PP
```

Comment:
whoa, how it was supposed to work?!

Reply:
If you removed the `-###` option, you would see that the command actually failed.